CMakeLists.txt

```diff
@@ -161,21 +161,18 @@ if (CMAKE_HOST_SYSTEM_PROCESSOR MATCHES "aarch64")
 else()
   set(LIBS_ARCH "x86_64")
   set(CONDA_LIBS
-      "libmkl_core.so"
-      "libmkl_gnu_thread.so"
-      "libmkl_intel_lp64.so"
+      "libiomp5.so"
+      "libmkl_core.so.2"
+      "libmkl_gnu_thread.so.2"
       "libmkl_intel_lp64.so.2"
-      "libmkl_intel_ilp64.so"
-      "libmkl_intel_ilp64.so.2"
-      "libmkl_intel_thread.so"
```

Comment:
the rest of the intel libraries are not needed?

Reply:
They are needed I just used version .2 instead of the generic one. It seems like new PyTorch version expects version 2.